docs/api/dt/countna.rst

```diff
@@ -0,0 +1,71 @@
+
+.. xfunction:: datatable.countna
+    :src: src/core/expr/head_reduce_unary.cc op_countna
+    :tests: tests/test-reduce.py
+    :cvar: doc_dt_countna
+    :signature: countna(cols)
+
+
+    Count the number of NA values for each column from `cols`.
+
+    Parameters
+    ----------
+    cols: Expr
+        Input columns.
+
+    return: Expr
+        f-expression having one row, and the same names and number of columns
+        as in `cols`. All the returned column stypes are `int64`.
+
+    except: TypeError
+        The exception is raised when one of the columns from `cols`
+        has a non-numeric and non-string type.
```

Comment:
We should be able to count NAs in all columns except `obj64`.

Reply:
Ack